src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerContentFacetImpl.java

```diff
@@ -190,6 +195,14 @@ protected Content doPutContent(final String path,
         false
     );
 
+    try {
+      asset.formatAttributes().clear();
```

Comment:
Note: This behaviour is something a future modifier would have to be aware of if for some reason we needed to add any derived attributes to this map. I would be inclined to only remove the composer supplied attributes or alternatively move this into a class like ComposerAttributeUpdate to make the behaviour more obvious.